### ISSUE TITLE
Update the XML schema to not accept offsets on metadata

### ISSUE THREAD

**Issue by duck2** [SymbiFlow/symbiflow-arch-defs#841]
Apparently, there are no X and Y offsets in metadata elements anymore, but the schema still accepts them:

```xsd
  <xs:complexType name="meta">
    <xs:simpleContent>
      <xs:extension base="xs:string">
        <xs:attribute name="name" type="xs:string" use="required"/>
        <xs:attribute name="xoffset" type="xs:int"/>
        <xs:attribute name="x_offset" type="xs:int"/>
        <xs:attribute name="yoffset" type="xs:int"/>
        <xs:attribute name="y_offset" type="xs:int"/>
      </xs:extension>
    </xs:simpleContent>
  </xs:complexType>
```

**Comment by mithro** [comment]
@litghost Probably knows about what happened here?

**Comment by litghost** [comment]
> @litghost Probably knows about what happened here?

The offset were never used in FASM, and upstream VTR devs identified that the offset logic was dead code, so it was removed.

**Comment by mithro** [comment]
Fixed in #872.
